packages/devextreme/js/__internal/ui/chat/chat.ts

```diff
@@ -29,6 +30,8 @@ const TEXTEDITOR_INPUT_CLASS = 'dx-texteditor-input';
 type Properties = ChatProperties & {
   title: string;
   showDayHeaders: boolean;
+  dayHeaderFormat: null | Format;
```

Comment:
JFYI: you will have conflict when https://github.com/DevExpress/DevExtreme/pull/28219 is merged